201-managed-application-with-customprovider/artifacts/ManagedAppZip/createUiDefinition.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+  "handler": "Microsoft.Azure.CreateUIDef",
+  "version": "0.1.2-preview",
+  "parameters": {
+    "basics": [],
+    "steps": [
+      {
+        "name": "applicationSettings",
+        "label": "Application Settings",
+        "subLabel": {
+          "preValidation": "Configure your application settings",
+          "postValidation": "Done"
+        },
+        "bladeTitle": "Application Settings",
+        "elements": [
+          {
+            "name": "funcname",
+            "type": "Microsoft.Common.TextBox",
+            "label": "Name of the function to be created",
+            "toolTip": "Name of the function to be created",
+            "visible": true
+          },
+          {
+            "name": "storagename",
+            "type": "Microsoft.Common.TextBox",
+            "label": "Name of the storage to be created",
+            "toolTip": "Name of the storage to be created",
+            "visible": true
+          }
+        ]
+      }
+    ],
+    "outputs": {
```

Comment:
add location to the outputs, use the resourceTypes property to filter the locations you need